docs/run-testing-framework.md

```diff
@@ -60,11 +60,25 @@ Remember, if you have changes on sample apps or the mocked server, you need to r
  
 #### 2.1.3 Build Image
 Please follow https://github.com/aws-observability/aws-otel-collector/blob/main/docs/developers/build-docker.md to build your image with the new component, and push this image to dockerhub, record the image link, which will be used in your testing.
- 
-### 2.2 Run in ECS
+
+#### 2.2 Run in EC2
```

Comment:
I'm a bit confused. Why did you move this up? It used to be 2.4. If you're going to move things around, then you need to fix the links to the sections in the README.

Reply:
Mostly for consistency with what being show [in the README](https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/README.md#2-run-testing-framework). Also fixed the links. Nice catch